pkg/machinery/mixins.go

```diff
@@ -48,14 +48,25 @@ type TemplateMixin struct {
 	IfExistsActionMixin
 
 	// TemplateBody is the template body to execute
-	TemplateBody string
+	TemplateBody    string
+	parseDelimLeft  string
+	parseDelimRight string
 }
 
 // GetBody implements Template
 func (t *TemplateMixin) GetBody() string {
 	return t.TemplateBody
 }
 
```

Comment:
Could we add comments to the new methods?
See that they are exported and the comments will be used to doc them.